pinot-common/src/main/java/org/apache/pinot/common/function/scalar/StringFunctions.java

```diff
@@ -18,6 +18,10 @@
  */
 package org.apache.pinot.common.function.scalar;
 
+import it.unimi.dsi.fastutil.objects.ObjectArrayList;
```

Comment:
No need to use fastutil here. We can use the java default ones

Reply:
updated